File: 01_Language/01_Functions/php/getimagesizefromstring.php
<?php

$img = 'test.gif';

// Open as a file
$size_info1 = getimagesize($img);
var_dump($size_info1);

// Or open as a string
$data       = file_get_contents($img);
$size_info2 = getimagesizefromstring($data);
var_dump($size_info2);

File: 01_Language/01_Functions/python/getimagesizefromstring.py
# coding: utf-8

from PIL import Image
import io


def getimagesizefromstring(imagedata):
    return Image.open(io.BytesIO(imagedata)).size


if __name__ == '__main__':
    with open('test.gif', mode='rb') as f:
        print(getimagesizefromstring(f.read()))
